File: TheDescent/TheDescent.cpp
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>

int main()
{

    while (true) {
        int tallestIndex = -1; // Reset each new loop
        int tallestHeight = -1; // Reset each new loop
        
        for (int i = 0; i < 8; i++) {
            int mountainH; // represents the height of index at i.
            std::cin >> mountainH; std::cin.ignore();
        
            if(tallestIndex != -1)
            {
                if(mountainH > tallestHeight)
                {
                    tallestIndex = i;
                    tallestHeight = mountainH;
                }
            }
            else
            {
                tallestIndex = i;
                tallestHeight = mountainH;
            }
        
        }
        
        std::cout << tallestIndex << std::endl;
    }
}

File: Easy/TheDescent/TheDescent.py
import sys
import math

while True:
    tallest_index = -1
    tallest_height = -1
    
    for i in range(8):
        mountain_h = int(input())  # represents the height of one mountain.

        if(tallest_height != -1):
            if(mountain_h > tallest_height):
                tallest_index = i
                tallest_height = mountain_h
        else:
            tallest_index = i
            tallest_height = mountain_h

    print(tallest_index)
